src/kernels/MFEMVectorFETimeDerivativeMassKernel.C

```diff
@@ -0,0 +1,21 @@
+#include "MFEMVectorFETimeDerivativeMassKernel.h"
+
+registerMooseObject("PlatypusApp", MFEMVectorFETimeDerivativeMassKernel);
+
+InputParameters
+MFEMVectorFETimeDerivativeMassKernel::validParams()
+{
+  InputParameters params = MFEMMassKernel::validParams();
+  params.addClassDescription("Adds the domain integrator to an MFEM problem for the bilinear form "
+                             "$(k \\dot{\\vec u}, \\vec v)_\\Omega$ "
+                             "arising from the weak form of the operator "
+                             "$k \\dot{ \\vec u}$.");
+  return params;
+}
+
+MFEMVectorFETimeDerivativeMassKernel::MFEMVectorFETimeDerivativeMassKernel(
+    const InputParameters & parameters)
+  : MFEMVectorFEMassKernel(parameters),
+    _var_dot_name(platypus::GetTimeDerivativeName(_test_var_name))
```

Comment:
Judging from the CodeCov warnings, you aren't actually doing any tests with this kernel. It would be good to add an example and/or a unit test.

Reply:
Agreed - these slipped in from the development of another example. I've removed them from this PR as I think they're out-of-scope here